docs/style_guide/README.md

```diff
@@ -1,49 +1,48 @@
-# Ballerina style guide
+# Ballerina Style Guide
 
-Ballerina Style Guide aims at maintaining a standard coding style among Ballerina community. Therefore, Ballerina IDE plugins and tools follow this guide when providing code formatting and other features. 
+The Ballerina Style Guide aims at maintaining a standard coding style among the Ballerina community. Therefore, Therefore, the code formatting tools are based on this guide.
 
-You can follow your own coding style when writing Ballerina source code and plugins and tools can be configured to match your coding style.
+You can follow your own coding style when writing Ballerina source code. Also, plugins and tools can be configured to match your coding style.
 
 ### Indentation and line length
 * Use four spaces (not tabs) for each level of indentation.
-* Keep the maximum length of a line as 120 characters. 
+* Keep the maximum length of a line to 120 characters. 
 
-> Note: tools and plugins should be able to configure to use tabs when indenting and change the number of maximum characters for line length.
+> **Note:** You can configure tools and plugins to use tabs when indenting and to change the number of maximum characters of the line length.
 
 ### Line spacing
 
 * Use only a single space to separate keywords, types, and identifiers. 
    
-    Dos
+    **Do's**
     ```ballerina
     public function getFullName() returns string {
         string fullName = "john doe";
         return fullName;
     }
     ```
-    Don'ts
+    **Don'ts**
     ```ballerina
     public   function   getFullName()   returns   string {
         string   fullName = "john doe";
         return     fullName;
     }
     ```
-  Few exceptions for this rule are
+  Few exceptions for this rule are;
```

Comment:
```suggestion
  Few exceptions for this rule are
```

Reply:
Fixed as "Few exceptions for this rule are:".